package.json

```diff
@@ -33,8 +33,8 @@
     "babel-eslint": "^10.1.0",
     "babel-plugin-dev-expression": "^0.2.2",
     "babel-plugin-transform-remove-imports": "^1.3.2",
-    "chalk": "^3.0.0",
-    "cheerio": "^1.0.0-rc.3",
+    "chalk": "^4.1.1",
```

Comment:
Major version bump but no significant API changes from 3.x, all the scripts using `chalk` check out just fine.